app/models/user.rb

```diff
@@ -12,6 +12,7 @@
 #  home_lat             :float
 #  home_lon             :float
 #  home_zoom            :integer          default(3)
+#  location_name        :string
```

Comment:
Why is it not `home_location_name`? You're sometimes referring to it as simply "home location", right?